emails/2023-12-22.md

```diff
@@ -0,0 +1,27 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+For most people around the world, next week is a holiday week. So, this will be my last email of 2023. Wishing you and yours a happy holidays.
```

Comment:
```suggestion
For most people around the world, next week is a holiday week. So, this will be my last email of 2023. Wishing you and yours a happy holiday.
```